app/apollo/resolvers/channel.js

```diff
@@ -598,6 +604,9 @@ const channelResolvers = {
           }
         } ) );
 
+        // Allow graphQL plugins to retrieve more information
+        context.pluginContext = {name: newVersionObj.name, uuid: newVersionObj.uuid};
```

Comment:
pluginContext needs to include details about created subscription(s), if created